README.md

```diff
@@ -1,12 +1,36 @@
 # Swarm Stack Updater
-The Swarm Stack Updater an automated tool built in shell that can be run periodically on a docker swarm to search for and identify any out of date stacks. If any changes are found, this tool automatically downloads, updates, and redeploys software back onto the swarm.
+The Swarm Stack Updater an automated tool built in shell that can be run periodically on a Docker Swarm to search for and identify any out of date stacks. If any changes are found, this tool automatically downloads, updates, and redeploys software back onto the swarm.
 
-This tool has been developed in conjuction with the University of Canterbury Computer Science Education Research Group as a University of Canterbury Sofware Engineering final year project. 
+This tool has been specifically enginneered to find and update full stacks by downloading Docker Compose files from GitHub. This was after finding tools such as [Watchtower](https://github.com/containrrr/watchtower) or [Shepard](https://github.com/djmaze/shepherd) which only aimed to update individual Docker containers by updating images. Using Compose files means whole application stacks can be updated and can allow for configuration changes as well.  
 
-## How to use:
 
-### Deploying Using Docker
-* First create a config file matching this description:
+## Requirements
+* Needs to be run in a docker swarm, this can either be locally or on a remove server.
+* The tool makes use of the Github API and Container Registry. You may have to update the script to work for other 
+repositories and container registries.
+* Website must have a status url that returns a JSON object that contains both 
+sha commit hash and a tag version.
+* ```{"VERSION_NUMBER": "3.10.0", "GIT_SHA": "a49a111d"}```
```

Comment:
```suggestion
  ```{"VERSION_NUMBER": "7.1.0", "GIT_SHA": "634d994d771fa23f65bc735bba32317ca71b374d"}`
  ```
```

Remove bullet, and show full hash.